general/administration/reverse-proxy.md

```diff
@@ -327,31 +327,41 @@ docker-compose.yml :
 version: '3'
 
 services:
-  proxy:
-    container_name: Traefik
-    image: traefik:alpine
-    command: --web --docker --docker.domain=${DOMAIN} --logLevel="INFO"
+  traefik:
+    container_name: traefik
+    hostname: traefik
+    image: traefik:traefik:v1.7.19
     networks:
       - traefik
     ports:
       - 80:80
       - 443:443
-    expose:
-      - 8080
+      - 8080:8080
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
       - ./traefik.toml:/traefik.toml
       - ./acme.json:/acme.json
-    restart: unless-stopped
+    restart: always
```

Comment:
Unless-stopped would make more sense to me for an example, so that it could be easily shut down if required. My Docker-Compose knowledge is very limited though.

Reply:
You're right, Unless-stopped might be a better option for people. Unfortunately, due to traefik shifting v2 to their main branch, you have to specify which v1.7 version you want. Traefik v1 is deprecated at this point but I don't think a v2 guide would be appropriate for this.

Reply:
Thanks for confirming the version part, I don't use Traefik myself so didn't realize they did that.

Reply:
fixed the restart policy, should be good to merge.